docs: add litestar, fix some grammar and spacing (#1550)
scalar/scalar: README.md

- Edit your Swagger files with a live preview
- Doesn’t look like it’s 2011

> 

Author: New admonition syntax: https://github.com/orgs/community/discussions/16925

### With Nuxt

You can easily run Scalar API References in Nuxt via the module:

```bash
npx nuxi module add @scalar/nuxt
```

> If you are using Nuxt server routes, you can enable scalar simply by enabling openAPI in the nitro

Author: ```suggestion
If you are using Nuxt server routes, you can enable scalar simply by enabling `openAPI` in the nitro
```

The API Reference package is written in Vue, that shouldn’t stop you from using

> it in React though! We have created a client side (untested on SSR/SSG) wrapper

Author: consider moving this out of and into some kind of warning admonition:

<details>
<summary>Details</summary>

<img width="515" alt="image" src="https://github.com/scalar/scalar/assets/45884264/b19f39dd-fcfc-4233-992b-70498595b9cf">

</details>

function App() {
  return (
    <ApiReferenceReact
      configuration = {
  {
    spec: {
      url: 'https://petstore3.swagger.io/api/v3/openapi.json',
    }
  ,
  }
}
  />
)
}

export default App
```

### With Next.js

> Our Next.js handler makes it easy to render a reference; just add it to an Api

Author: ```suggestion
Our Next.js handler makes it easy to render a reference; just add it to an API
```

function App() {
  return (
    <ApiReferenceReact

>       configuration = {

Reviewer: I think I’d prefer this to be intended. Wdyt?

Author: I reverted it :D

### Theme Prefix Changes

We've migrated our `--theme-*` CSS variables to `--scalar-*` to avoid conflicts with other CSS variables in
applications consuming the Scalar references or themes.
If you're injecting your custom CSS through the [`customCss`](#configuration) configuration option we will automatically
migrate your variable prefixes but display a warning in the console.

We recommend updating your theme variables as soon as possible:

- `--theme-*` → `--scalar-*`
- `--sidebar-*` → `--scalar-sidebar-*`

For a before and after example of an updated theme
see [`legacyTheme.css`](https://github.com/scalar/scalar/tree/main/packages/themes/src/fixtures/legacyTheme.css)
and [`updatedTheme.css`](https://github.com/scalar/scalar/tree/main/packages/themes/src/fixtures/updatedTheme.css)
in the [`@scalar/themes`](https://github.com/scalar/scalar/tree/main/packages/themes/) package.

## Community

> We are API nerds. You too? Let’s chat on Discord: <https://discord.gg/8HeZcRGPFS>

Author: `discord.gg/scalar`?

> We are API nerds. You too? Let’s chat on Discord: <https://discord.gg/8HeZcRGPFS>

Author: ```suggestion
We are API nerds. You too? Let’s chat on Discord: <http://discord.gg/scalar>
```
